one-acre-fund/ussd-sms
# Kenya enrollment fixes

* [SER-334](https://oneacrefund.atlassian.net/browse/SER-334) Kenya Enrollment/Just In time fixes(part 2)
    * updated maize bundle logic
    * updated eligibility to topUp/enroll check
    * added a back navigation


To test [here](https://telerivet.com/p/0c6396c9/service/SVc3c8d451e239d756/edit) 
**Test JiT enrollment:**

- enter a GL account number: **13983972**
- choose option 3 JiT Enrollment
- use an account number in the same group: **31217433** or **31217454** or **31217479**
- choose any product( can choose up to 3 products. a product ordered should not appear on the list next time. can only choose one quantity for maize bundles and can choose variety. back navigation also works)
- Finalize the order. ***the used account number can't order or top up the next time**

**To test TopUp**

- enter a GL account number: **13983972**
- choose option 4 JiT TopUp
- use an account number in the same group: **12122497** (only account enrolled in LR2021 can topUp and once)
- choose any product( can choose up to 3 products. a product ordered should not appear on the list next time. can only choose one quantity for maize bundles and can choose variety. back navigation also works)
- Finalize the order. ***the used account number can't order or top up the next time**
